tests/operations/test_cache.py

```diff
@@ -0,0 +1,200 @@
+from __future__ import annotations
+
+import hashlib
+import pathlib
+
+import emsarray
+import emsarray.operations.cache
+
+# Sha1
+empty_sha1_hash = "da39a3ee5e6b4b0d3255bfef95601890afd80709"
+attr_hash_lon = "86508f0e34ae9311b4dd4643e5afbf275eaf0052"
+str_hash = 'd79c41b2fb9cbfa7e81c951747609057691cb686'
+int_hash = '604b8984b112bf57de911f48a75ec769dd63411f'
+attr_hash_lat = "08d7bc76818a474e80847c26ed78d588374bfbc4"
+cache_key_hash_cf1d_sha1 = "414050a852498ee4b57154ead45b048756004cb5"
```

Comment:
This is all going to break terribly as soon as we release a new version of emsarray, because `emsarray.__version__` will change. Keeping the version in the hash is still essential, in case the geometry generation changes between versions, but this does present challenges...

A [pytest fixture](https://docs.pytest.org/en/stable/how-to/fixtures.html) scoped to this test module that [monkeypaches](https://docs.pytest.org/en/stable/how-to/monkeypatch.html) the `emsarray.__version__` attribute should do the trick. You can hardcode the version string to any arbitrary version just for these tests. That should keep the hashes stable for future releases.